src/js/elements/subindicator_filter/filter_controller.js

```diff
@@ -175,8 +175,9 @@ export class FilterController extends Component {
     shouldFiltersBeVisible() {
         const nonAggregatableGroups = this.model.dataFilterModel.nonAggregatableGroups;
         const defaultGroups = this.model.dataFilterModel.defaultFilterGroups;
+        const previouslySelectedFilterGroups = this.model.dataFilterModel.previouslySelectedFilterGroups;
 
-        if (nonAggregatableGroups.length <= 0 && defaultGroups.length <= 0 && this.model.dataFilterModel.availableFilters.length <= 0) {
+        if (nonAggregatableGroups.length <= 0 && defaultGroups.length <= 0 && previouslySelectedFilterGroups.length <= 0 && this.model.dataFilterModel.availableFilters.length <= 0) {
```

Comment:
Maybe assigning length to the constants could make the if statement a little neater. Not a big issue though

```
const previouslySelectedFilterGroups = this.model.dataFilterModel.previouslySelectedFilterGroups.length;
if (previouslySelectedFilterGroups <= 0) {...}
```